packages/data-context/src/sources/MigrationDataSource.ts

```diff
@@ -87,12 +87,11 @@ export class MigrationDataSource {
       throw Error('cannot get specs without a project path')
     }
 
-    const compFolder = await this.componentFolder()
     const intFolder = await this.integrationFolder()
 
-    const specs = await getSpecs(this.ctx.currentProject, compFolder || null, intFolder || null)
+    const specs = await getSpecs(this.ctx.currentProject, intFolder || null)
```

Comment:
Should we worry about users who have custom testSpec regex pattern in their <10.0 configuration? Currently the pattern(s) will be used to find the specs, whether or not they are in the integration folder: https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/specs.ts#L173

Also, why did we remove migrating the tests found in the component folder?

Reply:
Good catch, fixing this one up.

Reply:
Also re: migrating component tests, you were right, I'd misunderstood a requirement - the first step of the migration will **rename** them if you are using the defaults, eg `cypress/component/foo.spec.js` will become `cypress/component/foo.cy.js`. In the next step, we ask you to manually move the component specs to be colocated with their component. 

This part of the migration (specifically, for component tests) is really not ideal. Although you can opt to leave them in `cypress/component`, most people like to colocate their component tests. Anyway, that's a product/design decision beyond the scope of this PR - the vast majority of users are E2E, and the workflow works really well for those users.